livekit-android-sdk/src/main/java/io/livekit/android/room/Room.kt

```diff
@@ -483,6 +483,17 @@ constructor(
         localParticipant.handleSubscribedQualityUpdate(subscribedQualityUpdate)
     }
 
+    override fun onSubscriptionPermissionUpdate(update: LivekitRtc.SubscriptionPermissionUpdate) {
```

Comment:
Do we want to emit any application facing events when this happens? As a subscriber, you are not able to hear/see a track sounds like a bug. But the reason is because of the source of those tracks not giving you permissions. So, it feels like emitting an event and let the apps notify in whatever way is appropriate to the end user would be helpful.

Reply:
Oh yeah, we'll want those, good catch.